plugins/meson.build

```diff
@@ -26,7 +26,9 @@ if json_c_dep.found()
     'plugins/wdc/wdc-utils.c',
     'plugins/ymtc/ymtc-nvme.c',
     'plugins/zns/zns.c',
+    'plugins/sed/sed.c',
```

Comment:
If you use `subdir` below, you don't need to list the file here as well. The `sed/meson.build` file should list this file in this case.